README.md

```diff
@@ -0,0 +1,38 @@
+# Simple Node.js Demo
+
+This repository contains a simple Node.js application for demonstration purposes.
+
+## Dependency
+
+- [Node.js](https://nodejs.org/)
```

Comment:
Can you add instructions to add node.js?

Reply:
Can I perhaps add an article instead on how to download Node? It's pretty lengthy to put here.

Reply:
Okay, that will work.